ess/src/API/EMBC.ESS/Managers/Submissions/SubmissionsManager.cs

```diff
@@ -154,14 +154,18 @@ public async Task<VerifySecurityQuestionsResponse> Handle(VerifySecurityQuestion
                 }
             }
 
-            return await Task.FromResult(ret);
+            return ret;
         }
 
-        public async Task<bool> Handle(VerifySecurityPhraseQuery query)
+        public async Task<VerifySecurityPhraseResponse> Handle(VerifySecurityPhraseQuery query)
         {
             bool maskSecurityPhrase = false;
             var file = await evacuationRepository.Read(query.FileId, maskSecurityPhrase);
-            return await Task.FromResult(file.SecurityPhrase.Equals(query.SecurityPhrase));
+            VerifySecurityPhraseResponse ret = new VerifySecurityPhraseResponse
+            {
+                IsCorrect = file.SecurityPhrase.Equals(query.SecurityPhrase)
```

Comment:
I think the specs for this is to make it case insensitive, can you check the story?